crates/rolldown_testing/src/integration_test.rs

```diff
@@ -442,10 +442,12 @@ impl IntegrationTest {
     let cwd = bundler.options().cwd.clone();
     let dist_folder = cwd.join(&bundler.options().dir);
 
-    let is_output_cjs = matches!(bundler.options().format, OutputFormat::Cjs);
+    let is_expect_executed_under_esm = matches!(bundler.options().format, OutputFormat::Esm)
+      || (!matches!(bundler.options().format, OutputFormat::Cjs)
+        && matches!(bundler.options().platform, Platform::Browser));
```

Comment:
Wonder if how we truely simulate the browser environment. Or we just assume node+esm is enough.